esmvaltool/diag_scripts/autoassess/land_surface_soilmoisture/soilmoisture.py

```diff
@@ -128,7 +127,7 @@ def land_sm_top(run):
     # record provenance
     plot_file = "Autoassess soilmoisture metrics"
     caption = 'Autoassess soilmoisture MedAbsErr for {}'.format(str(seasons))
-    provenance_record = get_provenance_record(plot_file, caption, run)
+    provenance_record = get_provenance_record(caption, run)
     cfg = {}
     cfg['run_dir'] = run['out_dir']
     # avoid rewriting provenance when running the plot diag
```

Comment:
This looks a bit odd. @valeriupredoi Could you please review the autoassess changes and see if everything still works as expected?

Reply:
looks good in theory, I'll test run one recipe on JASMIN now

Reply:
See https://github.com/ESMValGroup/ESMValTool/pull/2296#pullrequestreview-753780133.